src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import classNames from 'classnames';
+import { useCallback } from 'react';
+import { Todo } from '../../types/Todo';
+import { FilterType } from '../../types/Filter';
+
+type Props = {
+  todos: Todo[];
+  filterType: string;
+  setFilterType: (value: string) => void;
+  removeTodo: (todoId: number) => void;
+};
+
+export const Footer: React.FC<Props> = ({
```

Comment:
Usually, mentors don't comment on the same mistakes in code for the same checklist rule